docs/advanced/api_examples.yml

```diff
@@ -245,7 +245,8 @@
     "get state datasets"
 
   doc: |
-    Get a list of all datasets in a deployment.
+    Get a list of all datasets in a deployment. The filesystem path where
+    the dataset is mounted will be included.
```

Comment:
It's the one thing that is fundamentally different between this endpoint and all the others that related to datasets. This is true regardless of whether we have two schemas or one.